package com.nt.vikas;

public class TryCatchFinally {
	String s1="Vikas";

	TryCatchFinally() throws InterruptedException{
			}
	
	
	/*public static void main(String[] args) {
		System.out.println(m1(3));

	}
	static int m1(int p) {
		while(p++ <=5) {
		try {
			System.out.println(10/0);
			return 10;
		}
		catch(ArithmeticException e) {
			System.out.println("In catch");
			//e.printStackTrace();
			}
		finally {
			System.out.println("In finally");
		
			}
		}
		System.out.println("After loop");
		return 20;
	}*/
	}
	 class Sample extends TryCatchFinally{

		Sample() throws InterruptedException {
			super();
			
		}
		
		
	}
